daemon/logger/journald/journald.go

```diff
@@ -48,10 +49,17 @@ func New(ctx logger.Context) (logger.Logger, error) {
 		name = name[1:]
 	}
 
+	// parse log tag
+	tag, err := loggerutils.ParseLogTag(ctx, "")
```

Comment:
Nope. I think it has to be user provided value so you can ignore logs from containers that don't have it set.